### DISCUSSION TITLE
Location problem: Microsoft Edge, system wide and site Location enabled.

### DISCUSSION THREAD

**Discussion by eurust** [community/community#60438] [GitHub Education]
### Hi everyone,

![image](https://github.com/community/community/assets/126270333/9ab454ef-15d5-4eee-80ee-b839da0a889b)

In the picture: my GitHub Education Student Package got rejected for location reasons, whilst I am using your **official operating system, your official browser** and have everything prepared correctly.

**Comment by samus-aran** [comment]
Hi @eurust 👋 

Have you been able to resolve the location issue on Edge or used an alternative browser ?

**Reply by Braggiouy** [reply]
I am having the same issue. Even though I have everything enabled, It does not recognize my campus being over the Canary Islands, whilst applying from the Canary Islands. Makes no sense at all. Tried to raise this issue, but no reply at all. 

I used Chorme and Firefox. Both giving me the same issue. 

Could it be that my campus is not correctly registered ? César Manrique, Canary Islands, Tenerife should be the correct one .@samus-aran

**Reply by samus-aran** [reply]
Hi @Braggiouy 

That sounds like a different issue. The Original Poster states they can not continue with the application due to the browser location access permissions being detected incorrectly. 

Yours sounds like you can continue with the application but your school is not on the map or if it is its in the wrong location? If that is the case then please raise a ticket with [Education support ](https://support.github.com/contact/education)using the dropdown "My selected school has incorrect or incomplete information e.g. domains or campus location". 👍 All applications must go through [GitHub Global Campus Student Application form](https://education.github.com/discount_requests/student_application). We are unable to provide manual reviews or upload links for individual applicants but we can review if the school details are accurate.

Hope this helps! Let us know how you get along 👍

**Reply by Braggiouy** [reply]
Appreciate your fast reply @samus-aran . I will proceed on rasing a ticket over the suggested support center. Thanks !

**Reply by eurust** [reply]
@samus-aran Yes, I have tried Google Chrome. But the problem is yet occurs.

**Reply by GCKKZoe** [reply]
I got exactly the same issue. Tried Chrome, Firefox, and Safari. Anyone knows how to solve it?

**Comment by Spixmacaw** [comment]
hii guys , want help

**Reply by samus-aran** [reply]
@Spixmacaw you need to be more specific. 👀  You are more likely to get a useful response if you are explicit about the feedback given to you from the rejected application submission. Do you have a screenshot of the rejection reasons? This might help someone give you a nudge in the right direction.

Note: Ensure that your query is relevant to the topic else please open up a separate discussion so that we can look at your query there. 👍
